### PR TITLE
Add missing CHANGELOG

### PR DESCRIPTION
**Pull request by rusty-snake**
#233

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,20 +14,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- `const-syscall` support for loongarch64 and m68k
 
 ### Changed
 
 #### Non-breaking
 - `get_api_sysrawrc` and `set_api_sysrawrc` can now be used with any API level.
 - `get_ctl_optimize` and `set_ctl_optimize` can now be used with any API level.
 - Rust 2021 Edition
+- updated `const-syscall` syscall definitions, including definitions for
+  `atomic_barrier`, `atomic_cmpxchg_32` and `getpagesize`
```
Comment (ManaSugi) [inline CHANGELOG.md:26]: Can these be move to the the existing changelog in the `Added` section as follows?

```
- `futex_waitv`, `set_mempolicy_home_node`, atmoic_barrier`,
  `atmoic_cmpxchg_32` and `getpagesize` syscalls for the `const-syscall` feature
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,20 +14,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
 #### breaking, caught by compiler
 - `ScmpFilterContext::{add,remove}_arch` return type changed to `Result<()>`.
   If you actually used the returned bool, call `ScmpFilterContext::is_arch_present` first.
 - Rename `ScmpFilterContext::new_filter` to `ScmpFilterContext::new`.
 - `ScmpFilterContext` methods support builder pattern.
+- IO-Safety for `ScmpFilterContext::export_*`.
```
Comment (ManaSugi) [inline CHANGELOG.md:33]: This shouldn't be included in this commit.

**Comment by rusty-snake** [comment]
Regarding failing tests, it's one of the most ugliest problems I debugged in the last 6 month.

**Comment by rusty-snake** [comment]
There are three tests that fail with libseccomp main + musl:

- `test_ensure_supported_api`
- `test_as_ptr`
- `test_ensure_supported_version`

even if I run only the `test_as_ptr` test (no other test), it fails with `signal: 11, SIGSEGV: invalid memory reference`.

**Comment by ManaSugi** [comment]
@rusty-snake 
It seems to have been solved when I re-run all jobs, but I don't have any idea why this problem happened.

**Comment by rusty-snake** [comment]
The thing is I can reproduce it local in podman. It *seems* to happen inside `seccomp_init`. Possible causes Rust-C-FFI, libseccomp, musl. Only the first one would be ours.
=> Something to keep an Eye on.

**Comment by ManaSugi** [comment]
@rusty-snake Thanks for the investigation.
Yes, we cannot ignore this problem, so I'll also take a look at this problem in detail. (In my experience, `musl-libc` always bothers me)

**Comment by rusty-snake** [comment]
@ManaSugi the most PRs can be merged from my side. When you have some time, you can go through them. musl+main keeps failing.

**Review by ManaSugi — APPROVED** [review]
LGTM Thanks @rusty-snake
